parser/grammar/sql-spec.txt

```diff
@@ -1123,7 +1123,7 @@ void numeric_primary():
   | character_value_expression()
 }
 
-void numeric_value_function() #FunctionCall:
+void numeric_value_function() #BuiltinFunctionCall(1):
```

Comment:
Curious what does (1) mean.

Reply:
1 means 1 child

Reply:
In this specific case it will be the ArgumentList child